gateway/gateway/entrypoints.py

```diff
@@ -25,7 +25,7 @@ def response_from_exception(self, exc):
         if isinstance(exc, self.expected_exceptions):
             if type(exc) in self.mapped_errors:
                 status_code, error_code = self.mapped_errors[type(exc)]
-            else:
+            else:  # pragma: no cover
```

Comment:
I can't really add a service test for this (as it means we have to throw an unmapped but expected exception)- so I guess this might be ok

tho maybe I should add a unit test just for this whole module...